.github/Contributing.md

```diff
@@ -8,36 +8,36 @@ By submitting code as an individual you agree that VATSIM UK can use your amendm
 
-To access Discord, you can visit https://www.vatsim.uk/discord and follow the registration instructions.  Once you've logged in, find the channel "#sector_file".
+To access Discord, you can visit https://www.vatsim.uk/discord and follow the registration instructions.  Once you've logged in, find the channel "#sector_file_development".
 
 # I want to contribute!
 
-If you wish to contribute to the UK Sector File project, there's many ways in which you can help out.
+If you wish to contribute to the UK Sector File project, there are many ways in which you can help out.
 
 ## Contributing to the data
 
-If you're just getting started with GitHub (and project contributions) then we suggest you take a look at issues marked with both the "up-for-grabs" and/or "good first issue" labels.  These issues will be of reasonable size and challenge while not being as overly complex as others and a good introduction for anyone who wants to start contributing to the project.  [This was inspired by an article by Ken C. Dodds](https://medium.com/@kentcdodds/first-timers-only-78281ea47455#.wior7p101).
+If you're just getting started with GitHub (and project contributions) then we suggest you take a look at issues marked with both the "up-for-grabs" and/or "good-first-issue" labels.  These issues will be of reasonable size and challenge while not being as overly complex as others and a good introduction for anyone who wants to start contributing to the project.  [This was inspired by an article by Ken C. Dodds](https://medium.com/@kentcdodds/first-timers-only-78281ea47455#.wior7p101).
 
 If you're comfortable with contributing to Open Source projects on GitHub please ensure you read our expectations for issue tracking, feature proposals and merge requests.
 
-**Please avoid** adding airac related commits until you've seen an issue created for them - anything else is fair game.
+**Please avoid** adding AIRAC related commits until you've seen an issue created for them - anything else is fair game.
 
 Please have a thorough read of the 1st-time Contributor's Guide: https://github.com/VATSIM-UK/UK-Sector-File/wiki/First-Time-Contributor's-Guide.
 
 ## Testing the sector file
 
-At present there isn't a publicly accessible auto-generation tool for the data within the repository.  We're working hard to make our current tool stable enough to release into the public domain so that open source contributions can be made to that, too, and others can use (and test) the latest development version.
+For each PR opened within the repository, the Sector File Compiler will generate a 'dev' version of the .sct, .ese and .rwy file for you to test out your changes. The compiler will also tell you if there are any errors. Click 'Details' in the **Checks** section of the Pull Request to check the errors.
```

Comment:
What about also linking to the compiler guide so people can use it as a development tool rather than just waiting until they send a PR to see it's broken? Also it's much easier once it's set up to use the compiler locally vs. manually editting the massive .sct file to get an SMR screenshot or whatever.